cornflow/schemas/execution.py

```diff
@@ -35,6 +35,8 @@ class ExecutionSchema(Schema):
     log_text = fields.Str(dump_only=True)
     log_json = fields.Nested(LogSchema, dump_only=True)
     finished = fields.Boolean(required=False)
+    state = fields.Int(validate=validate.Range(min=-1, max=1), required=False)
```

Comment:
this too, will need to be edited if we add errors. Maybe we should leave it open so it doesn't fail when we add a new error.

Reply:
I prefer to modify it every time we add a new error, as there is also a database constraint on the value so that we know which values we can have and which we cant. I can define it in the constant file, so that when we add a new error and its code it is automatically changed everywhere

Reply:
yes, better to have it as a defined constant in the `constants.py`. But then you have to change it in the model too and tie it to the `constant.py` file? seems complicated...